cli/src/cluster_query.rs

```diff
@@ -73,8 +73,7 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .validator(is_slot)
```

Comment:
`solana block` now dumps the latest confirmed block instead an error, which is a nice feature when watching for transaction fees.